client/my-sites/upgrades/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -275,18 +279,85 @@ class JetpackThankYouCard extends Component {
 	}
 
 	renderFeatures() {
-		let mappedFeatures;
-		if ( this.shouldRenderPlaceholders() ) {
-			mappedFeatures = this.renderFeaturePlaceholders();
-		} else {
-			mappedFeatures = this.getFeaturesWithStatus().map( this.renderFeature );
+		const { selectedSite } = this.props;
+
+		if ( selectedSite && ! selectedSite.canUpdateFiles ) {
+			return null;
 		}
 
-		return (
+		const mappedFeatures = this.shouldRenderPlaceholders()
+			? this.renderFeaturePlaceholders()
+			: this.getFeaturesWithStatus().map( this.renderFeature );
+		const features = (
 			<ul className="checkout-thank-you__jetpack-features">
 				{ mappedFeatures }
 			</ul>
 		);
+
+		if ( selectedSite && ! selectedSite.canManage() ) {
+			return (
+				<FeatureExample>
+					{ features }
+				</FeatureExample>
+			);
+		}
+
+		return features;
+	}
+
+	renderErrorDetail() {
+		const { translate, selectedSite } = this.props;
+		if ( ! this.isErrored() ) {
+			return null;
+		}
+
+		const reasons = utils.getSiteFileModDisableReason( selectedSite, 'modifyFiles' );
+		let reason;
+		if ( reasons && reasons.length > 0 ) {
+			reason = translate(
+				'We are unable to install the necessary plugins on your site due to lack of file permissions.'
+			);
+			this.trackConfigFinished( 'calypso_plans_autoconfig_error_filemod', { error: reason } );
+		} else if ( ! selectedSite.hasMinimumJetpackVersion ) {
+			reason = translate(
+				'We are unable to install the necessary plugins on your site because it has an older version of Jetpack.'
+			);
+			this.trackConfigFinished( 'calypso_plans_autoconfig_error_jpversion', {
+				jetpack_version: selectedSite.options.jetpack_version
+			} );
+		} else if ( ! selectedSite.isMainNetworkSite() ) {
+			reason = translate(
+				'We are unable to install the necessary plugins on your site because it is not the main network site.'
+			);
+			this.trackConfigFinished( 'calypso_plans_autoconfig_error_multisite' );
+		} else if ( selectedSite.options.is_multi_network ) {
+			reason = translate(
+				'We are unable to install the necessary plugins on your site because it is a multi-network sites.'
+			);
+			this.trackConfigFinished( 'calypso_plans_autoconfig_error_multinetwork' );
+		}
+
+		return (
+			<div>
+					<Card className="checkout-thank-you__jetpack-error-card">
+						<h3 className="checkout-thank-you__jetpack-error-heading">
+							{ translate( 'We had trouble setting up your plan' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'We had trouble setting up your plan.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).